docs/benchmark/ftlib_benchmark.md

```diff
@@ -0,0 +1,171 @@
+## Benchmark of FTlib AllReduce
+
+We perform experiments to test the performance of FTlib AllReduce in
+ElasticDL on three Kubernetes clusters -- a Minikube cluster, an on-premise CPU
+cluster managed by a tailored Kubernetes system, and a GPU cluster
+managed by the same tailored Kubernetes system. In those experiments,
+we adopt ResNet50 and MobileNetV2 models to test the performance of
+computation and communication. ResNet50 is a computation-intensive
+model and MobileNetV2 is a communication-intensive model.
+
+## Minikube Cluster
+
+We start a local Kubernetes cluster on macOS by the command:
+
+```bash
+minikube start --vm-driver=hyperkit --cpus 2 --memory 6144 --disk-size=50gb
+```
+
+The experiment parameters are:
+
+| Experiment parameter | Value    |
+| -- | --- |
+| Batch size | 64 |
+| Batches per task | 50 |
+| Images per task | 3220 |
+| Dataset | CIFAR-10 |
+| Input shape | (32, 32, 3) |
+| Output classes| 10 |
+| Worker resource| cpu=0.3,memory=2048Mi,ephemeral-storage=1024Mi|
+
+### ResNet50 with CIFAR-10 on a Minikube Cluster
+
+There are 214 parameter tensors in ResNet50 for CIFAR-10 dataset. And
+those tensor totally have 23,555,082 trainable parameters.
+
+| Workers |  computation/communication  |  Speed   |  Speedup Ratio |
+| -- | --------------------------- | -------- | ------ |
+| 1  |  -  | 3.1 images/s  |  1  |
+| 2  | 10: 1 | 5.65 images/s | 1.82 |
+
+### MobileNetV2 with CIFAR-10 on a Minikube Cluster
+
+There are 158 parameter tensors in ResNet50 for CIFAR-10 dataset. And
```

Comment:
ResNet50 => MobileNetV2

Please rephrase by using the ResNet50 model as the subject.